tests_e2e/orchestrator/lib/agent_test_suite.py

```diff
@@ -528,7 +550,7 @@ def _check_agent_log(self) -> bool:
             self.context.lisa_log.info("Checking agent log on the test node")
             log.info("Checking agent log on the test node")
 
-            output = self.context.ssh_client.run_command("check-agent-log.py -j")
+            output = self.context.ssh_client.run_command("$(get-agent-python) ~/bin/check-agent-log.py -j")
```

Comment:
get_distro() in version.py is not returning the expected values in some distros when using Pypy. I need to follow up that, and in the meanwhile I am using the system's python to execute this script.